parser/src/parser.cpp

```diff
@@ -28,6 +29,19 @@ po::options_description commandLineArguments()
   desc.add_options()
     ("help,h",
       "Prints this help message.")
+    ("workspace,w", po::value<std::string>()->required(),
+      "Path to a workspace directory. Project log files and local databases go "
+      "under this directory.")
+    ("name,n", po::value<std::string>()->required(),
+      "Project name.")
+    ("force,f",
+      "If the project already exists with the given name in the workspace, "
+      "then by default CodeCompass skips all parsing actions. Force parsing "
+      "removes all previous results from the workspace for the project.")
+    ("continue,c",
```

Comment:
I still have this problem if I run the program with the `-f` and `-c` options at the same time.